logs

```diff
@@ -24,41 +25,74 @@ class Logs
     @tail = in_args(args, '-t', '--tail')
     @data_to_get = (extract_from_args(args, '-n') || ENV.fetch('DATA_LENGTH', @tail ? '0' : '50000')).to_i
 
-    @marathon_name = args[0] 
-    @cmd = args[1] || 'stdout'
-    if marathon_name == '' || marathon_name == nil
-      display_usage
-      exit 0
+    if in_args(args, '-s', '--system-log')
+      @system_log = true
+      if args.any?
+        display_usage
+        exit 0
+      end
+    else
+      @marathon_name = args[0]
+      @cmd = args[1] || 'stdout'
+      if marathon_name == '' || marathon_name == nil
+        display_usage
+        exit 0
+      end
     end
   end
 
-  attr_reader :marathon_name, :cmd, :host, :data_to_get, :tail
+  attr_reader :marathon_name, :cmd, :host, :data_to_get, :tail, :system_log
 
   def run
-    begin
-      raise ApplicationNotFound if marathon_name == '' || marathon_name == nil
-      hosts = hosts_task_is_running_on(marathon_name)
-    rescue ApplicationNotFound
-      puts "Unable to find marathon task #{marathon_name}"
-      puts 'Valid task names are:'
-      puts available_apps.map { |a| " - #{a}" }.sort
-      exit 1
+    if system_log
+      output_system_log
+    else
+      output_app_log
     end
+  end
+
+  def output_system_log
+    hosts = get_hosts('marathon-consul/marathon-consul')
```

Comment:
Why does it need to look for this applications host?

Reply:
It needs an app to lookup the hostname - the apps endpoint doesn't include this inf.
The hostname in order to make the request. 
I have used the marathon-consul as I believe it is one of the first things to start and so was a good choice, happy to do this differently if you can think of a better approach. :)

Reply:
Can we use `mesos-proxy/web` instead, `marathon-consul` is not an application that others may be using.